UITests-2.xctestplan

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
Just a comment... Looking at the test results, looks like this test plan takes about 7 mins to run 44 tests and the other about 5, 36. I'm not familiar with the tests and divided them without knowing, but we could adjust the tests a bit more and have them running in about 6mins. Or even create a 3rd test plan and reduce the test time even more...